src/marina_pool.erl

```diff
@@ -175,6 +181,7 @@ start_node(<<A, B, C, D>> = RpcAddress) ->
 
     case shackle_pool:start(NodeId, ?CLIENT, ClientOptions, PoolOptions) of
         ok ->
+            marina_bucket:start_link(NodeId),
```

Comment:
is marina_bucket under some supervisor? what if it crashes

Reply:
even if marina_bucket crashes and restarts again, all State data in it is lost, could this be problematic?